source/Tutorials/Intermediate/Launch/Creating-Launch-Files.rst

```diff
@@ -3,64 +3,64 @@
   Tutorials/Launch-Files/Creating-Launch-Files
   Tutorials/Launch/Creating-Launch-Files
 
-Creating a launch file
+Crear un archivo de lanzamiento
 ======================
 
-**Goal:** Create a launch file to run a complex ROS 2 system.
+**Objectivo:** Crear un archivo de lanzamiento para correr un sistema complejo de ROS2.
 
-**Tutorial level:** Intermediate
+**Nivel del tutorial:** Intermedio
 
-**Time:** 10 minutes
+**Tiempo:** 10 minutos
 
 .. contents:: Contents
    :depth: 2
    :local:
 
-Prerequisites
+Prerequisitos
 -------------
 
-This tutorial uses the :doc:`rqt_graph and turtlesim <../../Beginner-CLI-Tools/Introducing-Turtlesim/Introducing-Turtlesim>` packages.
+Este tutorial usa los paquetes :doc:`rqt_graph y turtlesim <../../Beginner-CLI-Tools/Introducing-Turtlesim/Introducing-Turtlesim>`.
 
-You will also need to use a text editor of your preference.
+También deberá utilizar un editor de texto de su preferencia.
 
-As always, don’t forget to source ROS 2 in :doc:`every new terminal you open <../../Beginner-CLI-Tools/Configuring-ROS2-Environment>`.
+Como siempre, no se olvide de sincronizar las fuentes de ROS2 en :doc:`cada terminal que abra <../../Beginner-CLI-Tools/Configuring-ROS2-Environment>`.
 
-Background
+Antecedentes
 ----------
 
-The launch system in ROS 2 is responsible for helping the user describe the configuration of their system and then execute it as described.
-The configuration of the system includes what programs to run, where to run them, what arguments to pass them, and ROS-specific conventions which make it easy to reuse components throughout the system by giving them each a different configuration.
-It is also responsible for monitoring the state of the processes launched, and reporting and/or reacting to changes in the state of those processes.
+El sistema de lanzamiento en ROS2 es responsable de ayudar al usuario en la descripción de la configuración de su sistema y de ejecutarlo como fue descrito.
```

Comment:
Yo traduciría "The launch system" por "El sistema de launch", igual que "launchers" lo dejaría en inglés ("lanzadores" me suena regular)